reference/tls/cleartext-redirection.md

```diff
@@ -21,7 +21,38 @@ Ambassador Edge Stack exposes configuration for this in two ways:
 
 Typically, port-based redirection is the preferred method since it is simpler to manage and will work with all use cases. Redirecting based off of the `x-forwarded-proto` header requires an L7 load-balancer or proxy in front of Ambassador Edge Stack to set that header.
 
-## Port-based Redirection
+## Using the `Host` resource
+
+As of Ambassador API Gateway and Edge Stack 1.0.0, the [`Host`](/reference/host-crd) resource is used to configure how Ambassador handles cleartext on a domain. You can now configure Ambassador to deny, allow, or redirect cleartext to HTTPS on a domain using a single resource.
```

Comment:
I'm not a fan of including specific versions in the docs because it's easy to forget this and it starts looking dated. We'll version the docs at some point, but for now, I think it's better to just assume someone reading the docs is reading latest.